src/openapi.yaml

```diff
@@ -25,9 +25,9 @@ paths:
     post:
       operationId: userLogin
       tags:
-        - editor
-        - idempotent
+        - User
       security:
+        - authToken: []
```

Comment:
This should be an empty array, so that the user can log in.

Reply:
figured out a way to give it empty array that swagger editor does not moan about -- will have to see if Redoc.ly likes it too